src/utils/search/generic_search.jl

```diff
@@ -64,14 +64,14 @@ end
 
 "Create a search tree Node, derived from a parent by an action."
 function Node(state; parent = nothing, action = nothing, path_cost = 0.0)
-    depth = parent != nothing ? parent.depth + 1 : 0
+    depth = parent !== nothing ? parent.depth + 1 : 0
```

Comment:
What's this operator !== ?

Reply:
it's the new version supported by Julia: != -> !== and == -> ===